leabra7/projn.py

```diff
@@ -186,11 +187,18 @@ def __init__(self,
         self.pre = pre
         self.post = post
 
+        self.cos_diff_avg = 0.0
+
```

Comment:
careful of unnecessary white space (unless it is necessary and I'm not understanding the purpose haha)